lighthouse-core/runner.js

```diff
@@ -180,6 +180,12 @@ class Runner {
       // As entries can share a name, dedupe based on the startTime timestamp
     ].map(entry => /** @type {[number, PerformanceEntry]} */ ([entry.startTime, entry]));
     const timingEntries = Array.from(new Map(timingEntriesKeyValues).values());
+    for (const timing of timingEntries) {
+      // @ts-ignore - ignore readonly
+      timing.startTime = parseFloat(timing.startTime.toFixed(2));
```

Comment:
It looks like you may have to make a copy of these in order to do this. Real `PerformanceEntry`'s properties really do appear to be read only

Reply:
yeaup. 
![image](https://user-images.githubusercontent.com/39191/55111455-acbf1b00-5097-11e9-93b8-b4d425655fa6.png)

Reply:
done